app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -1137,6 +1137,20 @@ export default {
         cc.push(conversationContact);
       }
 
+      // Add each address in emailAttributes.to to the "cc" list, expect for specific conditions
+      // If the email is not a support or reply email, and is not the conversation contact's email, then add it to the "cc" list
+      if (Array.isArray(emailAttributes.to)) {
+        emailAttributes.to.forEach(email => {
+          if (
+            !email.includes('support') &&
```

Comment:
This is not a robust test for the email address of an inbox

Reply:
Do you have any suggestion on how this can be implemented?